options.go

```diff
@@ -18,10 +18,18 @@ type Options struct {
 	// message to the server
 	IdleTime time.Duration
 
+	// ReadTimeout is the maximum time between read events before the
+	// ReadTimeoutHandler is called
+	ReadTimeout time.Duration
```

Comment:
I think we need a default value for this, otherwise `<-time.After(0*time.Second)` would fire endlessly.

See: https://go.dev/play/p/gOlaqNI-UqT

Reply:
This does fire endlessly when ran locally:

```
when: 2022-06-14 16:38:50.354385 -0500 CDT m=+1.800189243
when: 2022-06-14 16:38:50.354388 -0500 CDT m=+1.800192336
when: 2022-06-14 16:38:50.354392 -0500 CDT m=+1.800196537
when: 2022-06-14 16:38:50.354394 -0500 CDT m=+1.800199192
when: 2022-06-14 16:38:50.354398 -0500 CDT m=+1.800203148
when: 2022-06-14 16:38:50.354401 -0500 CDT m=+1.800205735
when: 2022-06-14 16:38:50.354405 -0500 CDT m=+1.800209760
when: 2022-06-14 16:38:50.354425 -0500 CDT m=+1.800229379
when: 2022-06-14 16:38:50.354427 -0500 CDT m=+1.800231794
when: 2022-06-14 16:38:50.354429 -0500 CDT m=+1.800234150
when: 2022-06-14 16:38:50.354455 -0500 CDT m=+1.800259537
when: 2022-06-14 16:38:50.359297 -0500 CDT m=+1.805102074
```

Reply:
I set it to 60 seconds, which is longer than the 45 seconds we'll use in our implementation.